Matplotlib_common/plot_series.py

```diff
@@ -17,11 +17,13 @@ def plot_series(var_list, plot_iter, nplotx, nploty, axarr, xscaledict, yscaledi
     for file_name in file_names:
       print file_name, var
       series_file = open(file_name, "r")
-      my_times = read_my_var(series_file, "position")
+      #my_times = read_my_var(series_file, "position")
+      my_time = read_my_var(series_file, "time")
       my_res = read_my_var(series_file, var)
       
       # rescale time to hours
-      my_times = my_times / 3600.
+      #my_times = my_times / 3600. ##H
+      my_times = my__times / 60.  ##Minutes
```

Comment:
is my__times defined?
Also, my_times should be in hours for some cases, so let's keep it in hours by default.
If you want to have minutes for some particular case, make it case-specific.